docs/hooks/useUpdate.mdx

```diff
@@ -0,0 +1,65 @@
+---
+name: useUpdate
+menu: Hooks
+---
+
+import { useState } from 'react';
+import { Playground } from 'docz';
+import { useToggle, useUpdate } from '../../src';
```

Comment:
remove `useToggle` as it is no longer being used here. No dead variables in examples. it might mislead the user.